src/Workspaces/Core/Portable/PatternMatching/PatternMatcher.cs

```diff
@@ -52,7 +53,14 @@ internal abstract partial class PatternMatcher : IDisposable
             bool allowFuzzyMatching = false)
         {
             culture = culture ?? CultureInfo.CurrentCulture;
+            if (culture.Name == "en-US")
+            {
+                culture = CultureInfo.InvariantCulture;
```

Comment:
invariant culture gives us some nice fastpaths when doing String.indexOf.  For english this gives a nice speedup without affecting the quality of the results.